docs/Tags.md

```diff
@@ -1,29 +1,33 @@
+Tag fields are similar to full-text fields but use simpler tokenization and encoding in the index. The values in these fields cannot be accessed by general field-less search and can be used only with a special syntax.
 
 The main differences between tag and full-text fields are:
 
 1. We do not perform stemming on tag indexes.
 
-2. The tokenization is simpler: The user can determine a separator (defaults to a comma) for multiple tags, and we only do whitespace trimming at the end of tags. Thus, tags can contain spaces, punctuation marks, accents, etc. The only two transformations we perform are lower-casing (for latin languages only as of now), and whitespace trimming.
+2. The tokenization is simpler: The user can determine a separator (defaults to a comma) for multiple tags, and we only do whitespace trimming at the end of tags. Thus, tags can contain spaces, punctuation marks, accents, etc.
 
-3. Tags cannot be found from a general full-text search. If a document has a field called "tags" with the values "foo" and "bar", searching for foo or bar without a special tag modifier (see below) will not return this document.
+3. The only two transformations we perform are lower-casing (for latin languages only as of now) and whitespace trimming. Lower-case transformation can be disabled by passing CASESENSITIVE.
 
-4. The index is much simpler and more compressed: We do not store frequencies, offset vectors of field flags. The index contains only document IDs encoded as deltas. This means that an entry in a tag index is usually one or two bytes long. This makes them very memory efficient and fast.
+4. Tags cannot be found from a general full-text search. If a document has a field called "tags" with the values "foo" and "bar", searching for foo or bar without a special tag modifier (see below) will not return this document.
 
-5. An unlimited number of tag fields can be created per index, as long as the overall number of fields is under 1024.
+5. The index is much simpler and more compressed: We do not store frequencies, offset vectors of field flags. The index contains only document IDs encoded as deltas. This means that an entry in a tag index is usually one or two bytes long. This makes them very memory efficient and fast.
+
+6. An unlimited number of tag fields can be created per index, as long as the overall number of fields is under 1024.
 
 ## Creating a tag field
 
 Tag fields can be added to the schema in FT.ADD with the following syntax:
 
 ```
-FT.CREATE ... SCHEMA ... {field_name} TAG [SEPARATOR {sep}]
+FT.CREATE ... SCHEMA ... {field_name} TAG [SEPARATOR {sep}] [CASESENSITIVE]
 ```
 
 SEPARATOR defaults to a comma (`,`), and can be any printable ASCII character. For example:
 
+CASESENSITIVE can be specified to keep the original letter case.
```

Comment:
```suggestion
CASESENSITIVE can be specified to keep the original letters case.
```
This line is currently following `For example:` - did you intend for it to be here?